Clarification of DID creation (registration) (#88)
bcgov/trustdidweb: spec/specification.md

      keys are placed in the `updateKeys` item in [[ref: parameters]].
   3. The public key(s) of the authorization key pair(s) **MAY** be used in the
      [[ref: DIDDoc]] as well, but that is not required.
      
3. **Create the initial [[ref: DIDDoc]] for the DID**
   The [[ref: DIDDoc]] **MUST** contain the top level `id` item which **MUST** be the DID string from
   step 1, including the placement of the `{SICD}` placeholder for the [[ref: SCID]]. Other
   [[ref: DIDDoc]] verifications **SHOULD** be performed.
   
   All other absolute self-reference's to the DID in the [[ref: DIDDoc]] must use the form defined
   in step 1, with the identified placeholder for the [[ref: SCID]] (e.g., `did:tdw:{SCID}:example.com#key-1`,

>    `did:tdw:{SCID}:example.com:dids:issuer#key-1`, etc.). For the rest of the [[ref: DIDDoc]]

Reviewer: How about for the last line — “The [[ref: DIDDoc]] can contain any other content as deemed necessary by the [[ref: DID Controller]]."

>    `did:tdw:{SCID}:example.com:dids:issuer#key-1`, etc.). For the rest of the [[ref: DIDDoc]]

Author: ```suggestion
   `did:tdw:{SCID}:example.com:dids:issuer#key-1`, etc.). The [[ref: DIDDoc]] can contain any other content as deemed necessary by the [[ref: DID Controller]].
```

>    whatever content can be added as required.

Author: ```suggestion
 ```

>    1. Note: The placeholders (the string `{SCID}`) **MUST** be in every place in the [[ref: DIDDoc]] where

Reviewer: Singular “placeholder” in this spot.

>    1. Note: The placeholders (the string `{SCID}`) **MUST** be in every place in the [[ref: DIDDoc]] where

Author: ```suggestion
   1. Note: The placeholder (the string `{SCID}`) **MUST** be in every place in the [[ref: DIDDoc]] where
```

   the SCID is to be placed.
   
   
4. **Generate a preliminary DID Log Entry** (input JSON array)

>    The [[ref: DID log entriy]] is an input JSON array that when completed contains the following items:

Reviewer: Typo: “entriy"

>    The [[ref: DID log entriy]] is an input JSON array that when completed contains the following items:

Author: ```suggestion
   The [[ref: DID log entry]] is an input JSON array that when completed contains the following items:
```
